package.json

```diff
@@ -21,12 +21,21 @@
   "devDependencies": {
     "@babel/core": "^7.18.6",
```

Comment:
- [ ] I cannot start application by running 'npm start', this command should open the document on the browser and update it whenever you make changes, so you don't have to run `npm run build` and refresh page every time. All you need to do is updating package.json file and add `start` script: 
```json
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```